src/Ryujinx.Common/Utilities/ValueFormatUtils.cs

```diff
@@ -0,0 +1,196 @@
+        {
+            TimeSpan returnTimeSpan = TimeSpan.Zero;
+
+            var valueSplit = timeSpanString.Split(", ");
+            if (valueSplit.Length > 1)
+            {
+                var dayPart = valueSplit[0].Split("d")[0];
+                if (int.TryParse(dayPart, out int days))
+                {
+                    returnTimeSpan = returnTimeSpan.Add(TimeSpan.FromDays(days));
+                }
+            }
+
+            if (TimeSpan.TryParse(valueSplit.Last(), out TimeSpan parsedTimeSpan))
+            {
+                returnTimeSpan = returnTimeSpan.Add(parsedTimeSpan);
+            }
+
+            return returnTimeSpan;
+        }
+
+        /// <summary>
+        /// This method formats a <see cref="DateTime"/> so it can be displayed in the UI.
+        /// Only used as a canonical way to format the LastPlayed value.
+        /// </summary>
+        /// <param name="utcDateTime">The <see cref="DateTime"/> to be formatted. This is expected to be UTC-based.</param>
+        /// <param name="culture">The <see cref="CultureInfo"/> that's used in formatting. Defaults to <see cref="CultureInfo.CurrentCulture"/>.</param>
+        /// <returns>A formatted string that can be displayed in the UI.</returns>
+        public static string FormatDateTime(DateTime? utcDateTime, CultureInfo culture = null)
+        {
+            culture ??= CultureInfo.CurrentCulture;
+
+            if (!utcDateTime.HasValue)
+            {
+                // In the Avalonia UI, this is turned into a localized version of "Never" by LocalizedNeverConverter.
+                return "Never";
+            }
+
+            return utcDateTime.Value.ToLocalTime().ToString(culture);
+        }
+
+        /// <summary>
+        /// This method parses a string that was used in the UI into a <see cref="DateTime"/>.
+        /// </summary>
+        /// <param name="dateTimeString">The string representing a <see cref="DateTime"/>.</param>
+        /// <returns>A <see cref="DateTime"/> object. If the input string couldn't be parsed, <see cref="DateTime.UnixEpoch"/> is returned.</returns>
+        public static DateTime ParseDateTime(string dateTimeString)
+        {
+            if (!DateTime.TryParse(dateTimeString, CultureInfo.CurrentCulture, out DateTime parsedDateTime))
+            {
+                return DateTime.UnixEpoch;
+            }
+
+            return parsedDateTime;
+        }
+
+        /// <summary>
+        /// Formats a number of bytes to a human-readable string with adjustable units.
+        /// </summary>
+        /// <param name="size">The file size in bytes.</param>
+        /// <param name="forceUnit">Formats the passed size value as this unit, bypassing the automatic unit choice.</param>
+        /// <returns>A human-readable file size string.</returns>
+        public static string FormatFileSize(long size, FileSizeUnits forceUnit = FileSizeUnits.Auto)
+        {
+            double _base = 1024;
```

Comment:
```suggestion
            double base = 1024;
```

Local variables are not prefixed with an underscore.

Reply:
C# doesn't allow you to name a variable `base`, so I put an underscore on it.

Reply:
Renamed all `_base` variables to `SizeBase`.